arviz/utils.py

```diff
@@ -601,3 +601,28 @@ def get_coords(data, coords):
         except KeyError as err:
             raise KeyError("Error in data[{}]: {}".format(idx, err))
     return data_subset
+
+
+def credible_interval_warning(credible_interval, hpd_interval):
+    """Helper method to warn that credible interval will be deprecated"""
+
+    warnings.warn(
+        (
+            "Keyword argument credible_interval has been deprecated "
+            "Please replace with hpd_interval"
+        ),
+    )
+
+    if isinstance(credible_interval, str) and credible_interval == "auto":
```

Comment:
@OriolAbril Does this address your concerns?

Reply:
I'm also going to change this to an exception because it raises an exception further down and the warning is up top.

![image](https://user-images.githubusercontent.com/7213793/81412806-43626600-90f9-11ea-891a-a0160e3529e4.png)

Reply:
It is the other way around, what used to happen was:
* `credible_interval="auto"` -> use rcParams
* `credible_interval=None` -> hide hpd line (only implemented in plot_posterior because everywhere else (like hpd or plot_forest) it did not really make sense to hide 
* otherwise use value, has to be float between 0 and 1 ...